src/APIActionCreator.es6.js

```diff
@@ -181,9 +207,19 @@ export default class APIActionCreator extends ActionCreator {
 				request.headers = Object.assign({}, defaultHeaders.headers || {}, request.headers || {});
 			}
 
+			// If the user has defined a default base URL, append the route to the
+			// base URL if, and only if, the route itself is not a fully qualified
+			// URL.
+			let requestRoute = request.route ? request.route : route;
```

Comment:
I'll fix this in another diff. route should never be a thing in createRequest (as per docs)